### ISSUE TITLE
Patching / Merging Pomicons with other fonts

### ISSUE THREAD

**Issue by karanr** [gabrielelana/pomicons#2]
Hi,
I am using https://github.com/ryanoasis/vim-webdevicons, PragmataPro (PragmataPro for Powerline Plus Nerd File Types.ttf) patched for airline and nerdtree.
I tried merging Pomicons.ttf with PragmataPro for Powerline Plus Nerd File Types.ttf using fontforge and was (unsuccessful) several errors.
Is there any way for Pomicons.ttf to be merged with Powerline Plus Nerd File Types.ttf, or any of the other patched fonts in vim-webdevicons?
thanks.

**Comment by gabrielelana** [comment]
I don't know, we can ask @ryanoasis about it, I'll be glad to give permission or a suitable licence if needed

**Comment by ryanoasis** [comment]
I can try to help out if you like. I am by no means a fontforge expert. I just built a patcher with the python fontforge module. 

Were you doing this with the gui? What were the errors? Did the glyphs show up?

I have had errors before in fontforge and still have merging work in some cases, I think it depends on the severity of the errors.

**Comment by gabrielelana** [comment]
@ryanoasis sorry, I didn't mean that, I was wandering if you are willing to include this font in your project

**Comment by ryanoasis** [comment]
@gabrielelana Ah okay I see. Looks like the glyphs have to be in the `u+E000` through `u+E00A` range only?

I did see at least 1 font I use have a few characters in that range but the others all have that range free.

I would either add them to my core or add a flag to add these additional glyphs. I think it depends on how many fonts have characters in that range.

**Comment by gabrielelana** [comment]
@ryanoasis yep, it's the ["Private Use Area" block](http://en.wikipedia.org/wiki/Private_Use_Areas) so it should be available in most general use fonts. But take into consideration that when you copy pomicons (or any other icon font) into a base font you can always copy the glyphs wherever you want, as long as you document where you put them it should be fine. I have a [similar project](https://github.com/gabrielelana) where I do that exporting [maps](https://github.com/gabrielelana/awesome-terminal-fonts/blob/master/build/pomicons-regular.sh) that bind symbol names to their unicode code point 

Anyway, if you want to integrate pomicons in your project do ahead, I'll be glad

**Comment by ryanoasis** [comment]
@gabrielelana Right, cool I think I know what i'm going to do thanks. I will probably keep them in their original locations also. I remember seeing your awesome-terminal-fonts project before. :+1: 

@karanr , @gabrielelana I will try to post again on this thread my progress and updates

**Comment by ryanoasis** [comment]
@karanr @gabrielelana feel free to give any of these a try: https://github.com/ryanoasis/nerd-filetype-glyphs-fonts-patcher/tree/0.3.0/patched-fonts

FYI: very rough and not tested

**Comment by landfillbaby** [comment]
One thing I've just noticed: [Input Mono](https://input.djr.com/) stores its alternate "single-storey" lowercase `a` glyph at `U+E00A`, the same as `EXTERNAL_INTERRUPTION`. I'm not sure if anyone actually uses that glyph though as IMO it looks too much like lowercase `o`.
And `EXTERNAL_INTERRUPTION` is a lot like `U+26A1` `⚡︎` which is included in Input Mono anyway.
